docs/source/cli_help/github.rst

```diff
@@ -66,6 +60,12 @@ a GitHub API token with ``repo`` scope is required, see https://github.com/setti
 
   Sets the base of the current branch's PR to upstream (parent) branch, as seen by git machete (see ``git machete show up``).
 
+``sync``:
+
+  Synchronizes with the remote repository: deletes untracked managed branches with no downstream branch, deletes unmanaged branches and also checks out open PRs for the current user associated with the Github token,
```

Comment:
One last (I hope thing): I'd rather reorganize this into an (un?)ordered list for easier reading!

Reply:
Also... the word `delete` isn't correct here ‼️ 

```
            elif github_subcommand == "sync":
                machete_client.delete_untracked(opt_yes=False)
                machete_client.delete_unmanaged(opt_yes=False)
```

Reply:
(nvm, all good)